initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -559,26 +559,24 @@ stop_component() {
 
     eval $stopcmd
 
+    unlock ${LOCKPATH}
+    removePid ${PIDPATH}
+
     RESULT=0
-    local counter=0
-    while [[ $RESULT -eq 0 && $counter -le 20 ]]; do
+    local waittime=30
```

Comment:
Increasing the wait time to 600 seconds if compType = dali is a quick fix that won't break anything else.  And we'll be able to revisit the issue later.  I just want to get this fixed for 5.2.4.